src/SecTester.Scan/CI/CiServer.cs

```diff
@@ -1,6 +1,125 @@
+  public static CiServer CodeShip { get; } = new("CODESHIP", "Codeship");
+  public static CiServer Drone { get; } = new("DRONE", "Drone");
+  public static CiServer Dsari { get; } = new("DSARI", "dsari");
+  public static CiServer Eas { get; } = new("EAS", "Expo Application Services");
+  public static CiServer Gerrit { get; } = new("GERRIT", "Gerrit");
+  public static CiServer GithubActions { get; } = new("GITHUB_ACTIONS", "GitHub Actions");
+  public static CiServer GitLab { get; } = new("GITLAB", "GitLab CI");
+  public static CiServer GoCd { get; } = new("GOCD", "GoCD");
+  public static CiServer GoogleCloudBuild { get; } = new("GOOGLE_CLOUD_BUILD", "Google Cloud Build");
+  public static CiServer Heroku { get; } = new("HEROKU", "Heroku");
+  public static CiServer Hudson { get; } = new("HUDSON", "Hudson");
+  public static CiServer Jenkins { get; } = new("JENKINS", "Jenkins");
+  public static CiServer LayerCi { get; } = new("LAYERCI", "LayerCI");
+  public static CiServer Magnum { get; } = new("MAGNUM", "Magnum CI");
+  public static CiServer Netlify { get; } = new("NETLIFY", "Netlify CI");
+  public static CiServer NeverCode { get; } = new("NEVERCODE", "Nevercode");
+  public static CiServer ReleaseHub { get; } = new("RELEASEHUB", "ReleaseHub");
+  public static CiServer Render { get; } = new("RENDER", "Render");
+  public static CiServer Sail { get; } = new("SAIL", "Sail CI");
+  public static CiServer Screwdriver { get; } = new("SCREWDRIVER", "Screwdriver");
+  public static CiServer Semaphore { get; } = new("SEMAPHORE", "Semaphore");
+  public static CiServer Shippable { get; } = new("SHIPPABLE", "Shippable");
+  public static CiServer Solano { get; } = new("SOLANO", "Solano CI");
+  public static CiServer SourceHut { get; } = new("SOURCEHUT", "Sourcehut");
+  public static CiServer Strider { get; } = new("STRIDER", "Strider CD");
+  public static CiServer TaskCluster { get; } = new("TASKCLUSTER", "TaskCluster");
+  public static CiServer TeamCity { get; } = new("TEAMCITY", "TeamCity");
+  public static CiServer Travis { get; } = new("TRAVIS", "Travis CI");
+  public static CiServer Vercel { get; } = new("VERCEL", "Vercel");
+  public static CiServer Woodpecker { get; } = new("WOODPECKER", "Woodpecker");
+  public static CiServer XcodeCloud { get; } = new("XCODE_CLOUD", "Xcode Cloud");
+  public static CiServer XcodeServer { get; } = new("XCODE_SERVER", "Xcode Server");
+
+  private readonly string _constant;
+  private readonly string _name;
+  private readonly int _hashcode;
+
+  public string Constant => _constant;
+  public string Name => _name;
+
+  private CiServer(string constant, string name)
+  {
+    _constant = constant;
+    _name = name;
+    _hashcode = StringComparer.OrdinalIgnoreCase.GetHashCode(_constant);
+  }
+
+  public override int GetHashCode()
+  {
+    return _hashcode;
+  }
+
+  public override string ToString()
+  {
+    return _constant;
+  }
```

Comment:
It is not clear to me why don't show humanized branded name

```suggestion
  public override string ToString() => _name;
```